<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportPreview</class>
 <widget class="QMainWindow" name="ReportPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Print Preview</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../reportmaster/src/res.qrc">
    <normaloff>:/images/view-page-multi-32.png</normaloff>:/images/view-page-multi-32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionFirst_Page"/>
   <addaction name="actionPrevious_Page"/>
   <addaction name="actionNext_Page"/>
   <addaction name="actionLast_Page"/>
   <addaction name="separator"/>
   <addaction name="actionSingle_Page"/>
   <addaction name="actionDouble_Pages"/>
   <addaction name="actionAll_Pages"/>
  </widget>
  <action name="actionPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="text">
    <string>Excel</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/print-32.png</normaloff>:/images/print-32.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPage_Setup">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/page-setup-32.png</normaloff>:/images/page-setup-32.png</iconset>
   </property>
   <property name="text">
    <string>Page Setup</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/fit-width-32.png</normaloff>:/images/fit-width-32.png</iconset>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="actionFit_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/fit-page-32.png</normaloff>:/images/fit-page-32.png</iconset>
   </property>
   <property name="text">
    <string>Fit Page</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/zoom-in-32.png</normaloff>:/images/zoom-in-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/zoom-out-32.png</normaloff>:/images/zoom-out-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionPotrait_Mode">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/layout-portrait-32.png</normaloff>:/images/layout-portrait-32.png</iconset>
   </property>
   <property name="text">
    <string>Potrait Mode</string>
   </property>
  </action>
  <action name="actionLandscape_Mode">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/layout-landscape-32.png</normaloff>:/images/layout-landscape-32.png</iconset>
   </property>
   <property name="text">
    <string>Landscape Mode</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/go-first-32.png</normaloff>:/images/go-first-32.png</iconset>
   </property>
   <property name="text">
    <string>First Page</string>
   </property>
  </action>
  <action name="actionPrevious_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/go-previous-32.png</normaloff>:/images/go-previous-32.png</iconset>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="actionNext_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/go-next-32.png</normaloff>:/images/go-next-32.png</iconset>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/go-last-32.png</normaloff>:/images/go-last-32.png</iconset>
   </property>
   <property name="text">
    <string>Last Page</string>
   </property>
  </action>
  <action name="actionSingle_Page">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/view-page-one-32.png</normaloff>:/images/view-page-one-32.png</iconset>
   </property>
   <property name="text">
    <string>Single Page</string>
   </property>
  </action>
  <action name="actionDouble_Pages">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/view-page-sided-32.png</normaloff>:/images/view-page-sided-32.png</iconset>
   </property>
   <property name="text">
    <string>Double Pages</string>
   </property>
  </action>
  <action name="actionAll_Pages">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/view-page-multi-32.png</normaloff>:/images/view-page-multi-32.png</iconset>
   </property>
   <property name="text">
    <string>All Pages</string>
   </property>
  </action>
  <action name="actionAbout_Report_Master">
   <property name="text">
    <string>About Report Master</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../reportmaster/src/res.qrc">
     <normaloff>:/images/qtlogo-64.png</normaloff>:/images/qtlogo-64.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionGenerate_Report_From_SQL">
   <property name="text">
    <string>Generate Report From SQL</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../reportmaster/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
